pkg/skaffold/survey/config.go

```diff
@@ -60,7 +60,8 @@ type config struct {
 }
 
 func (s config) isActive() bool {
-	return s.expiresAt.IsZero() || s.expiresAt.After(time.Now())
+	return s.expiresAt.IsZero() ||
+		(s.startsAt.Before(time.Now()) && s.expiresAt.After(time.Now()))
```

Comment:
Funny: I found this really hard to reason through — we must think of time differently!

I'd have written this as "we're after the start and before the end":

    time.Now().After(s.startsAt) && time.Now().Before(s.expiresAt)

though there's a small boundary condition there that your formulation deals with 😆